newIDE/app/src/MainFrame/index.js

```diff
@@ -377,8 +379,13 @@ const MainFrame = (props: Props) => {
     project: gdProject,
     fileMetadata: ?FileMetadata
   ): Promise<State> => {
-    const { eventsFunctionsExtensionsState } = props;
-    if (fileMetadata) preferences.insertRecentProjectFile(fileMetadata);
+    const { eventsFunctionsExtensionsState, getStorageProvider } = props;
+
+    if (fileMetadata)
+      preferences.insertRecentProjectFile({
+        fileMetadata,
+        storageProviderName: getStorageProvider().internalName,
```

Comment:
@4ian `getStorageProvider` always returns "Empty", am I missing something? Apart from this, everything should be working fine.

Reply:
Mmm that's very strange - I think you have done everything that is needed in your PR. This must be a bug in `master`, let me check.

Reply:
It's indeed a bug in master. I'm busy now but will push a fix. Your PR should be good to go then :)